src/request.ts

```diff
@@ -1147,7 +1164,7 @@ class DatastoreRequest {
       };
     }
 
-    datastore.auth.getProjectId((err, projectId) => {
+    datastore.auth.getProjectId((err: Error|null|undefined, projectId: string|null|undefined) => {
```

Comment:
You really shouldn't need to explicitly define the types here.

Reply:
Agreed, it's just that I get an intense amount of errors if I don't. I'll change it and submit regardless